examples/launch_height_benchmark_local.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
I am not sure we need this example, unless `benchmarking/nursery/launch_local/hpo_main.py` cannot be used

Reply:
If `hpo_main.py` cannot be used, we can keep this one here, but remark this is mostly used to help remote launching.

Reply:
This scirpt importa and specifies new custom benchmark that is not present in hpo_main.py this is wh we need it. It can be useful for launching local jobs with local backend using benchmarking